docs/index.html

```diff
@@ -628,34 +628,16 @@ <h4 class="headline-3">Module: Typographie</h4>
 			</div>
 		</div>
 	</section>
-  
-	<section class="grid">
```

Comment:
Why did you remove grid? It's still used right?

Reply:
The whole description was duplicated and introduced at line 740 as well. Maybe we should move the grid explanations to the top of the documentation?

Reply:
Grid explanations moved to the top